.scalafmt.conf

```diff
@@ -20,5 +21,9 @@ project.excludeFilters = [
 fileOverride {
   "glob:**/scala-3*/**" {
     runner.dialect = scala3
+    rewrite.scala3.convertToNewSyntax = true
+    rewrite.rules = [RedundantBraces]
+    rewrite.scala3.removeOptionalBraces = yes
```

Comment:
To keep it in line with the rest of the codebase we might even need to disable optional braces, but not sure.